nistats/first_level_model.py

```diff
@@ -0,0 +1,561 @@
+        then its mask will be used. If no mask is given,
+        it will be computed automatically by a MultiNiftiMasker with default
+        parameters.
+
+    target_affine: 3x3 or 4x4 matrix, optional
+        This parameter is passed to nilearn.image.resample_img. Please see the
+        related documentation for details.
+
+    target_shape: 3-tuple of integers, optional
+        This parameter is passed to nilearn.image.resample_img. Please see the
+        related documentation for details.
+
+    low_pass: False or float, optional
+        This parameter is passed to nilearn.signal.clean.
+        Please see the related documentation for details.
+
+    high_pass: False or float, optional
+        This parameter is passed to nilearn.signal.clean.
+        Please see the related documentation for details.
+
+    smoothing_fwhm: float, optional
+        If smoothing_fwhm is not None, it gives the size in millimeters of the
+        spatial smoothing to apply to the signal.
+
+    memory: string, optional
+        Path to the directory used to cache the masking process and the glm
+        fit. By default, no caching is done. Creates instance of joblib.Memory.
+
+    memory_level: integer, optional
+        Rough estimator of the amount of memory used by caching. Higher value
+        means more memory for caching.
+
+    standardize : boolean, optional
+        If standardize is True, the time-series are centered and normed:
+        their variance is put to 1 in the time dimension.
+
+    signal_scaling: bool, optional,
+        If True, fMRI signals are scaled to the mean value of scaling_axis
+        Incompatible with standardize (standardize=False is enforced when\
+        signal_scaling is True).
+
+    scaling_axis: int or tuple, optional,
+        Axis used for signal scaling. 0 refers to mean scaling each voxel with
+        respect to time, 1 refers to mean scaling each time point with respect
+        to all voxels and (0, 1) refers to scaling with respect to voxels and
+        time, which is known as grand mean scaling.
+
+    noise_model : {'ar1', 'ols'}, optional
+        The temporal variance model. Defaults to 'ar1'
+
+    verbose : integer, optional
+        Indicate the level of verbosity. By default, nothing is printed.
+
+    n_jobs : integer, optional
+        The number of CPUs to use to do the computation. -1 means
+        'all CPUs', -2 'all CPUs but one', and so on.
+
+    minimize_memory : boolean, optional
+        Gets rid of some variables on the model fit results that are not
+        necessary for contrast computation and would only be useful for
+        further inspection of model details. This has an important impact
+        on memory consumption. True by default.
+
+    subject_id : string, optional
+        id of the subject of this first_level_model instance. Will be used
+        to create subject specific memory cache if caching allowed.
```

Comment:
I don't think we need this feature. This signatures are becoming horrible, so I'd like to avoid that parameter.